RecoTracker/LST/plugins/alpaka/LSTModulesDevESProducer.cc

```diff
@@ -29,7 +29,7 @@ class LSTModulesDevESProducer : public ESProducer {
 
   static void fillDescriptions(edm::ConfigurationDescriptions &descriptions);
 
-  std::optional<SDL::modulesBuffer<alpaka_common::DevHost>> produce(const TrackerRecoGeometryRecord &iRecord);
+  std::optional<SDL::modulesBuffer<SDL::Dev>> produce(device::Record<TrackerRecoGeometryRecord> const& iRecord);
```

Comment:
is the ` CopyToDevice<SDL::modulesBuffer<alpaka_common::DevHost>>` needed after this change?

Reply:
That's a good point. It should no longer be needed.